cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/client/AzureClient.java

```diff
@@ -984,6 +985,12 @@ public DiskEncryptionSetInner createDiskEncryptionSet(String diskEncryptionSetNa
         });
     }
 
+    public boolean checkKeyVaultExistence(String resourceGroupName, String vaultName) {
+        Vault keyVault = azure.vaults()
+                .getByResourceGroup(resourceGroupName, vaultName);
+        return (keyVault != null) ? true : false;
```

Comment:
Please wrap the method body in `return handleAuthException(() -> {...})` like done for `createDiskEncryptionSet()`. This is to ensure that the case when the Azure credential app has expired gets handled correctly.